{
	"schema_version": "1.6.0",
	"id": "DLA-1519-1",
	"modified": "2024-01-09T20:31:20.596080Z",
	"published": "2018-09-25T00:00:00Z",
	"related": [
		"CVE-2017-1000158",
		"CVE-2018-1000802",
		"CVE-2018-1060",
		"CVE-2018-1061"
	],
	"summary": "python2.7 - security update",
	"affected": [
		{
			"package": {
				"ecosystem": "Debian:8",
				"name": "python2.7",
				"purl": "pkg:deb/debian/python2.7?arch=source"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.7.9-2+deb8u2"
						}
					]
				}
			],
			"versions": [
				"2.7.9-2",
				"2.7.9-2+deb8u1"
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/debian-osv/dla-osv/DLA-1519-1.json"
			}
		}
	]
}
